.env-testnet-sample

```diff
@@ -62,3 +70,11 @@ LTC_RPC_PASS=sparkswapltc
 #
 EXTERNAL_BTC_ADDRESS=sample.ip.address
 EXTERNAL_LTC_ADDRESS=sample.ip.address
+
+# Docker compose files
+# docker-compose by default will look for the docker-compose.yml file and apply the docker-compose.override.yml file if
+# it exists. If we specify the COMPOSE_FILE variable, all compose files joined by the COMPOSE_PATH_SEPARATOR will
+# be applied and used
+#
+COMPOSE_PATH_SEPARATOR=:
+COMPOSE_FILE=docker-compose.yml:docker-compose.bitcoind.yml:docker-compose.litecoind.yml:docker-compose.override.yml
```

Comment:
Is it possible for us to use other variables to build up this variable? e.g. have a USE_BUNDLED_BITCOIND variable that if set to true, includes docker-compose.bitcoind.yml in the path, but otherwise doesn't.